### PR TITLE
Adding zendesk honeypot

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?
This pr adds a simple honeypot for some dork that's spamming our Zendesk tickets. If they keep persisting after this we'll look into something more advanced like [this](https://developers.google.com/recaptcha/)

#### How should this be reviewed?
Submit the Zendesk form, everything should work normally. Then try again, but fill-out the value of the honeypot input. 

honey

#### Any background context you want to provide?
I was originally going to implement our own shitty captcha, but this is a bit easier and doesn't impact the user at all.

### CODE REVIEW

**Review by DFurnes — APPROVED** [review]
Works for me! 🍯

File: lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module [hunk 1]
```diff
@@ -72,6 +72,11 @@ function dosomething_zendesk_form($form, &$form_state, $entity = NULL)  {
     $ip_address = ip_address();
   }
 
+  $form['honeypot'] = [
```
Comment (weerd) [inline lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module]: Maybe don't call it honeypot? Might the bot know to ignore a field labeled as such?

Comment (DFurnes) [reply lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module]: Could name it `not_honeypot`. :trollface:

Comment (weerd) [reply lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module]: `bots_please_fill_this_out`?

**Review by weerd — APPROVED** [review]
🍯 🐝

File: lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module (hunk 1 above)
Comment (itsjoekent) [reply lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module]: I prefer `not_honeypot` 😆 one second

Comment (itsjoekent) [reply lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module]: `git commit -m "Renamed to be more stealthy."`

**Comment by itsjoekent** [comment]
[(4) As Hannah, I don't want folks to be able to submit a Zendesk ticket if they're a spamming robot](https://trello.com/c/TgXIP0gj/725-4-as-hannah-i-dont-want-folks-to-be-able-to-submit-a-zendesk-ticket-if-theyre-a-spamming-robot)

**Pull request merged**